cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -230,20 +268,28 @@ def _validate_curie(self, term_id: str, column_name: str, curie_constraints: dic
 
         :rtype None
         """
-
-        # If there are forbidden terms
-        if "forbidden" in curie_constraints and term_id in curie_constraints["forbidden"]:
-            self.errors.append(f"'{term_id}' in '{column_name}' is not allowed.")
-            return
-
         # Check that term id belongs to allowed ontologies
-        self._validate_curie_ontology(term_id, column_name, curie_constraints["ontologies"])
+        is_valid_term = self._validate_curie_ontology(term_id, column_name, curie_constraints["ontologies"])
 
         # If the term id does not belong to an allowed ontology, the subsequent checks are redundant
-        if self.errors:
+        if not is_valid_term:
```

Comment:
nit: `is_valid_term` is not used anywhere else.
```suggestion
        if not self._validate_curie_ontology(term_id, column_name, curie_constraints["ontologies"]):
```